frontend/src/v5/ui/controls/chip/chip.styles.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Chip as ChipMui, ChipProps, ChipTypeMap } from '@mui/material';
+import { ElementType } from 'react';
+import styled, { css } from 'styled-components';
+
+type ChipVariants = ChipProps['variant'] | 'label' | 'label-outlined';
+
+export type ChipType<T extends ElementType = ChipTypeMap['defaultComponent']> = Omit<ChipProps<T>, 'variant'> & {
+	variant?: ChipVariants,
+	colour?: string,
```

Comment:
Option doesn't work (I wish it did), because it takes the value and uses it with your Mui theme. For instance if you pass in "primary" it will use "theme.palette.primary.contrastText" to get the text colour. So if you try to pass in a value that doesn't exist as a category in your Mui palette it crashes. 

`Uncaught TypeError: Cannot read properties of undefined (reading 'contrastText')`